src/redux/features/categories/categorySlice.js

```diff
@@ -0,0 +1,11 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categorySlice = createSlice({
+  name: 'categories',
+  initialState,
+});
+export default categorySlice.reducer;
```

Comment:
- [ ] It is required that when `check status` button is clicked, it displays  `Under construction` text. At this moment, that requirement has not been implemented. Kindly attend to it.

[app-2.webm](https://user-images.githubusercontent.com/26597432/230061317-0696ae1c-b320-4c1b-92a6-47b644b2fc0c.webm)